txScraper.py

```diff
@@ -24,7 +24,7 @@
 for row in table.findAll("tr"):
     for col in row.findAll("td"):
         for countyLink in col.findAll("a", href = True):
-            url = 'http://comptroller.texas.gov/propertytax/administration/pvs/findings/2013f/{}'.format(countyLink['href'])
+            url = 'https://comptroller.texas.gov/propertytax/administration/pvs/findings/2013f/{}'.format(countyLink['href'])
```

Comment:
OK, I've raised a bug about this : https://github.com/chaconne7/txPropertyValueScraper/issues/3